spec/models/generic_file_spec.rb

```diff
@@ -625,4 +625,156 @@
       end
     end
   end
+
+  let(:public_permission_double) { double("Hydra::AccessControls::Permission") }
+  let(:gv_black_photo_sub_collection_double) { double("Collection") }
+  let(:gv_black_collection_double) { double("Collection") }
+  let(:random_doi) { "10.18131/g3-nqgv-4863" }
+  let(:blank_doi) { "" }
+  let(:public_record) { GenericFile.new(doi: []) }
+  let(:private_record) { GenericFile.new(doi: []) }
+
+  # if record is publc, has a doi, and is either in the gv black photograph sub collection or is not in the greater gv black collection
+  describe "#unexportable?" do
+    before do
+      allow(public_permission_double).to receive(:agent).and_return(GenericFile::PUBLIC_PERMISSION)
+
+      allow(gv_black_photo_sub_collection_double).to receive(:map).and_return([GenericFile::GV_BLACK_PHOTOGRAPH_SUB_COLLECTION_ID])
```

Comment:
This is not the gv black photo sub collection, it's the collections the considered generic file is part of, no?
```suggestion
      allow(file_collections).to receive(:map).and_return([GenericFile::GV_BLACK_PHOTOGRAPH_SUB_COLLECTION_ID])
```

Reply:
Good point. The way I think of things in my head does not always translate to the code, ha.